src/Hedgehog/Script.fsx

```diff
@@ -35,7 +36,7 @@ Property.print <| property {
 //
 
 Property.print <| property {
-    let! xs = Gen.list Gen.int
+    let! xs = Gen.list <| Gen.bounded ()
```

Comment:
Nitpick: I could have used a type annotation here to make it clear we're asking for random integers. Same for a few other places below where I used `Gen.bounded ()`.